obstacle_stop_planner_refine/include/obstacle_stop_planner/obstacle_point_cloud.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OBSTACLE_STOP_PLANNER__OBSTACLE_POINT_CLOUD_HPP_
+#define OBSTACLE_STOP_PLANNER__OBSTACLE_POINT_CLOUD_HPP_
+
+#include <memory>
+
+#include <rclcpp/logger.hpp>
+#include <sensor_msgs/msg/point_cloud2.hpp>
+#include "pcl/point_types.h"
+#include "pcl_conversions/pcl_conversions.h"
+#include "pcl/point_cloud.h"
+#include "pcl/common/transforms.h"
+#include "pcl/filters/voxel_grid.h"
+#include "tf2_ros/transform_listener.h"
+#include "autoware_planning_msgs/msg/trajectory.hpp"
+#include "obstacle_stop_planner/vehicle.hpp"
+
+namespace motion_planning {
+
+class ObstaclePointCloud
+{
+public:
+  explicit ObstaclePointCloud(rclcpp::Logger logger);
```

Comment:
Do you want to copy logger? Should be const reference?

Reply:
I choose copy because `rclcpp::get_logger()` returns rclcpp::Logger class.

Reply:
I'm sorry, I couldn't understand. :cry: Could you explain a bit more detail?

Reply:
Sorry, I had wrong understanding. Fixed in 5b8e5e1